src/BeaniesList.js

```diff
@@ -1,8 +1,17 @@
-import BeanieBaby from './BeanieBaby.js';
+import { Link } from 'react-router-dom';
 
 export default function BeaniesList({ beanieBabies }) {
   return (
     <div className='beanies'>
-      {/* render out the beanie babies as a list */}
+      {
+        beanieBabies.map((BeanieBaby, i) =>
+          <Link key={BeanieBaby.title + i + BeanieBaby.id} to={`/BeanieBabies/${BeanieBaby.id}`}>
```

Comment:
the creation of this link, div, h3, p, and img is happening in BeanieBaby.js - and can be called here instead